purchase.adoc

```diff
@@ -388,288 +517,533 @@ Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation o
         "<https://purchase.izettle.com/purchases/v2?limit=10&descending=true&lastPurchaseHash=1485858080407AOjsYOefEeax5pDdmnaGXw>; rel=\"next\""
   ]
 }
-```
+----
+--
+
+'''
+
+====
+
+==== Fetch purchase by id
+
+`GET /purchase/v2/{purchaseUuid}`
+
+[%collapsible]
+====
+Returns a purchase with given UUID.
+
+*Parameters*
+[cols="15%,10%,10%,10%,55%"]
+|===
+|Name|Type|In|Required/Optional|Description
+
+|purchaseUuid|string|path|required|Unique id of the purchase as UUID within iZettle system.
+|===
+
+*Responses*
 
-### Get purchase details
+`Status: 200 OK` - Successful, response json is returned. +
+`Status: 404 Not found` - Purchase with given id not found. +
+`Status: 401 Unauthorized` - You were not authorized to execute this operation. +
+`Status: 429 Too Many Requests` - You hit the rate limit. Retry request later.
 
-`GET /purchase/v2/{purchaseUUID}`
+*Examples*
 
-#### Permissions required
-`READ:PURCHASE`
+* Fetching a specific purchase.
++
+--
+_Request:_
 
-purchaseUUID:: The UUID of the purchase
+`GET /purchase/v2/589c2de6-3ee4-11eb-9e70-a2ac075b17fd`
```

Comment:
Is there a request for listing all purchases without details to find the needed purchaseUuid?

Reply:
No, there is only one endpoint for listing purchases (with all the details).
However a purchase can have a reference to another purchase/purchases in form of `UUID`.